Nodejs-Bug-Bounty-Security-Fund.md

```diff
@@ -0,0 +1,34 @@
+# Node.js Bug Bounty/Security Fund
+
+The TSC maintains a LFX Crowdfunding account in order to accept
+the project component of bug bounties paid out by the
+[Hackerone](https://www.hackerone.com/) bug bounty program.
+
+The funds in the account are intended to be used by the TSC
+to encourage contributions to the project with respect to
+security and in particular the handling of vulnearabilities.
+
+This document outlines the use this account.
+
+## Account details
+
+The [account](https://crowdfunding.lfx.linuxfoundation.org/initiative/578a541a-4e7e-47a2-99b9-6cbf49b00c20)
+is associated with and LFX account. The user id and password
```

Comment:
"is associated with and LFX account" Is that supposed to be "an LFX"?  You use "a LFX" above. Pick one and stick with it.

Reply:
```suggestion
is associated with and Linux Foundation Crowdfunding account. The user id and password
```